_data/tools.yml

```diff
@@ -1323,6 +1326,16 @@
     Supports inheritance with discriminators, Jackson annotations and custom interceptors.
   v3: true
 
+- name: fetch-openapi
```

Comment:
Seems to only support OAS2 which is against best practice now OAS3.1 is out.

Reply:
Yeah, that should be updated.